src/utils/displayFunctions.ts

```diff
@@ -48,20 +66,45 @@ export const makeDisplayFunctions = (brandToInfo: Map<Brand, BrandInfo>) => {
     return stringifyRatio(ratio, getDecimalPlaces, placesToShow);
   };
 
-  const displayAmount = (amount: Amount, placesToShow?: number) => {
+  const displayAmount = (
+    amount: Amount,
+    placesToShow?: number,
+    localeFormat: boolean = false,
+    usdFormat: boolean = false,
```

Comment:
multiple interdependent booleans calls for an enum.
```
format?: 'usd' | 'locale' | undefined
```

Reply:
Nice, done